.changeset/fair-tips-deliver.md

```diff
@@ -3,4 +3,6 @@
 "@aws-amplify/ui-react": patch
```

Comment:
would this be considered an additive change? 
```suggestion
"@aws-amplify/ui-react": minor
```

Reply:
Good catch! Think `patch` is okay cause it's the internal exports but if public `minor` for sure